src/components/Button/Button.vue

```diff
@@ -120,6 +123,25 @@ export default {
 			type: String,
 			required: true,
 		},
+
+		/**
+		 * If provided, a tooltip will be displayed upon hovering the button.
+		 */
+		tooltipText: {
+			type: String,
+			default: 'tooltiptest',
+		},
+
+		/**
+		 * If you want more fine tuning over the tooltip behavior, you can pass
+		 * in a full "options' object instead of a tooltip string.
+		 * check https://github.com/Akryum/v-tooltip#other-options for more
+		 * information
+		 */
+		tooltipOptions: {
+			type: Object,
+			default: undefined,
+		},
```

Comment:
Why not just let the dev use v-tooltip on the component itself?

Reply:
You've got a point sir, shall I remove this?

Reply:
I think so yes :)